src/components/Home.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Card from '../components/Card'
+
+const Wrapper = styled.div`
+  display: grid;
+  justify-items: center;
+  grid-gap: 30px;
+  grid-auto-flow: center;
+  overflow-y: scroll;
+`
+
+export default class Home extends Component {
+  createCards() {
+    return this.props.teaserData.map(this.renderSingleCard)
+  }
+
+  renderSingleCard = teaserInfo => {
```

Comment:
You already have this function in your Favorites screen, I'd just import it here. Or the other way around. But never write a function twice ;).